modules/ROOT/partials/integrations/angular-tech-ref.adoc

```diff
@@ -128,15 +128,18 @@ include::partial$misc/get-an-api-key.adoc[]
 [[cloudchannel]]
 === `+cloudChannel+`
 
-*Default value:* `{productmajorversion}`
+*Type:* `+String+`
+
+*Default value:* `'{productmajorversion}'`
 
-*Possible values:* `{productmajorversion}-stable`, `{productmajorversion}-testing`, `{productmajorversion}-dev`
+*Possible values:* `{productmajorversion}-stable`, `{productmajorversion}-testing`, `{productmajorversion}-dev`, `'{productminorversion}'`
```

Comment:
I believe the other values should be wrapped in ' as well
```suggestion
*Possible values:* `'{productmajorversion}-stable'`, `'{productmajorversion}-testing'`, `'{productmajorversion}-dev'`, `'{productminorversion}'`
```

Reply:
Good catch, I really should have just copy/pasted the whole line instead of just the last part.